torch/csrc/autograd/python_torch_functions_manual.cpp

```diff
@@ -670,6 +678,7 @@ static PyObject * THPVariable__disable_functionalization(PyObject *self, PyObjec
 {
   HANDLE_TH_ERRORS
   c10::impl::tls_set_dispatch_key_included(at::DispatchKey::Functionalize, false);
+  c10::impl::tls_set_dispatch_key_included(at::DispatchKey::FunctionalizeAddBackViews, false);
```

Comment:
Isn't this going to prevent clean nesting?
In particular, if the python API should be a context manager, I would expect that it will re-set the state that was there before.
But maybe that's not considered here? If so, should we just forbid enabling it if it is already enabled?

Reply:
Yeah you're right, I wasn't thinking too much about the nesting case. Maybe I'll just forbid it now until there's a need.